cmd/main.go

```diff
@@ -30,7 +30,7 @@ import (
 
 func main() {
 
-	// Init command line flags to clear vendored packages' one, especially in init()
+	// Init command line flags to clear vendorred packages' one, especially in init()
```

Comment:
this should be vendored. See https://golang.org/src/README.vendor for the example.

Reply:
> this should be vendored. See https://golang.org/src/README.vendor for the example.

fixed